sdk/tables/azure-data-tables/README.md

```diff
@@ -126,18 +137,19 @@ The Azure Data Tables client library for Python allows you to interact with each
 use of a dedicated client object.
 
 ### Clients
-Two different clients are provided to to interact with the various components of the Table Service:
-1. [TableServiceClient](https://aka.ms/azsdk/python/tables/docs) -
+Two different clients are provided to interact with the various components of the Table Service:
+1. `TableServiceClient` -
     this client represents interaction with the Azure account itself, and allows you to acquire preconfigured
     client instances to access the tables within. It provides operations to retrieve and configure the account
     properties as well as query, create, and delete tables within the account. To perform operations on a specific table,
     retrieve a client using the `get_table_client` method.
-2. [TableClient](https://aka.ms/azsdk/python/tables/docs) -
+2. [`TableClient`][table_client_docs] -
```

Comment:
this still isn't rendering. Also if you're linking to the table client docs can you also do so for table service client. In the future in vs code, you can type ctrl-shift-v when you're on a md file, and it'll show a tab with everything rendered. You can also double check PRs etc by viewing the md file

Reply:
I don't know where I was trying to go. I don't have any documents on the clients specifically.